src/fetchers/workspace-fetcher.js

```diff
@@ -39,8 +39,7 @@ export default class WorkspaceFetcher {
       dest: this.dest,
       package: {
         ...pkg,
-        _uid: '',
-        version: '0.0.0',
+        _uid: (pkg && pkg.version) || '',
```

Comment:
I needed to do another change to be able to remove the checks. Please review again.